### ISSUE TITLE
测试集准确率0.95，但是写前向测试代码测得结果很差

### ISSUE THREAD

**Issue by ligaoyuan-potato** [JinghuiZhou/awesome_face_antispoofing#29]
model_path = r'../data/model/71.pth'

model = getattr(models, 'MyresNet34')().eval()
if torch.cuda.is_available():
    model = model.cuda()

    model.load(model_path)
else:

    model.load(model_path, map='cpu')

transformer = transforms.Compose([transforms.ToTensor()])#, transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])

def val(img):
    img = maxcrop(img)
    img = transformer(img)
    if torch.cuda.is_available():
        img = img.cuda()
    img = img.view(1, *img.size())
    with torch.no_grad():
        val_input = Variable(img)

    model.eval()
    score = model(val_input)

    outputs = torch.softmax(score, dim=-1)
    preds = outputs[0].to('cpu').detach().numpy()

    return preds

if __name__ == '__main__':
    path = r'../data/img/val/y'
    filelist = os.listdir(path)
    for file in filelist:
        image = os.path.join(path, file)
        img = Image.open(image).convert('RGB')
        res = val(img)
        print(res)
